openwisp_utils/tests.py

```diff
@@ -76,3 +77,79 @@ def stopTestRun(self):
 class TimeLoggingTestRunner(DiscoverRunner):
     def get_resultclass(self):
         return TimeLoggingTestResult
+
+
+class redirect_stdout(object):
```

Comment:
All these three functions, i.e., `redirect_stdout`, 'redirect_stderr` and `redirect_any_output` are similar to one another. Is there any way we can inherit them from one base function

Reply:
Yes, we can create a base class with `__call__` 

where we write code like:

```
if (self.stderr):
   sys.stderr = self.stderr
```
and inherit different classes out of with different `__init__()` 

This should work. Should I make an example?

Reply:
Yes, please it would be helpful

Reply:
Sure, feel free to update them in the meanwhile and please add the docs so that it's easier for me to use them! :smile: